forge-gui/res/cardsfolder/upcoming/brave_the_wilds.txt

```diff
@@ -0,0 +1,9 @@
+Name:Brave the Wilds
+ManaCost:G
+Types:Sorcery
+K:Bargain
+A:SP$ Animate | ValidTgts$ Land.YouCtrl | TgtPrompt$ Select target land you control | Power$ 3 | Toughness$ 3 | Types$ Creature,Elemental | Duration$ Permanent | SubAbility$ DBChangeZone | Condition$ Bargain | Keywords$ Haste | SpellDescription$ If this spell was bargained, target land you control becomes a 3/3 Elemental creature with haste that's still a land.
```

Comment:
> Some instant and sorcery spells require additional targets if they're bargained. You ignore those targeting requirements if those spells aren't bargained, and you can't bargain those spells unless you can choose the appropriate targets.

Reply:
So it needs a CheckSvar Count$Bargain or something like that ?

Reply:
No you need to work with TargetMin
(there should be some with kicker like that)